x/gov/keeper/deposit.go

```diff
@@ -187,17 +192,28 @@ func (keeper Keeper) ChargeDeposit(ctx context.Context, proposalID uint64, destA
 
 	// burn the cancellation fee or sent the cancellation charges to destination address.
 	if !cancellationCharges.IsZero() {
-		// get the distribution module account address
-		distributionAddress := keeper.authKeeper.GetModuleAddress(disttypes.ModuleName)
+		// get the pool module account address
+		poolAddress := keeper.authKeeper.GetModuleAddress(poolModuleName)
 		switch {
 		case destAddress == "":
 			// burn the cancellation charges from deposits
 			err := keeper.bankKeeper.BurnCoins(ctx, keeper.authKeeper.GetModuleAddress(types.ModuleName), cancellationCharges)
 			if err != nil {
 				return err
 			}
-		case distributionAddress.String() == destAddress:
-			err := keeper.distrKeeper.FundCommunityPool(ctx, cancellationCharges, keeper.ModuleAccountAddress())
+		case poolAddress.String() == destAddress:
+			amount := make([]*basev1beta1.Coin, len(cancellationCharges))
+			for i, coin := range cancellationCharges {
+				amount[i] = &basev1beta1.Coin{
+					Denom:  coin.Denom,
+					Amount: coin.Amount.String(),
+				}
+			}
+			msg := pooltypes.MsgFundCommunityPool{
+				Amount:    amount,
+				Depositor: keeper.ModuleAccountAddress().String(),
+			}
+			_, err := proto.Marshal(&msg)
```

Comment:
i think we need to pass this into the router? dont think we need to marshal here